README.md

```diff
@@ -9,8 +9,21 @@ Each of these Terraform projects are to be layered on top of each other, running
 
 ### [0. bootstrap](./0-bootstrap/)
 
-This stage executes the [CFT Bootstrap module](https://github.com/terraform-google-modules/terraform-google-bootstrap) which bootstraps an existing GCP organization, creating all the required GCP resources & permissions to start using the Cloud Foundation Toolkit (CFT).
-This includes; projects, service accounts and a Terraform state bucket. After executing this step, you will have the following structure:
+This stage executes the [CFT Bootstrap module](https://github.com/terraform-google-modules/terraform-google-bootstrap) which bootstraps an existing GCP organization, creating all the required GCP resources & permissions to start using the Cloud Foundation Toolkit (CFT). This includes:
+- The `cft-cloudbuild` project, which contains:
+  - Cloud Build implementation
+  - Cloud Source Repository
+  - Build pipeline
+  - Other resources
+- The `cft-seed` project, which contains:
+  - Terraform state bucket
+  - KMS configuration to encrypt the state bucket's content
```

Comment:
This is actually for use in builds if people wish to encrypt files / secrets in their pipelines